src/test/java/data/domain/CarTest.java

```diff
@@ -0,0 +1,26 @@
+package data.domain;
+
+import static org.assertj.core.api.AssertionsForInterfaceTypes.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class CarTest {
+
+
+    @Test
+    @DisplayName("자동차 이동 테스트 - 4 보다 크면 전진한다.")
```

Comment:
```suggestion
    @DisplayName("자동차 이동 테스트 - " + Car.THRESHOLD  + " 보다 크면 전진한다.")
```

조건이 변경 됐을 때 해당 테스트의 DisplayName을 신경쓰지 않으면 더 좋겠어요 :)

Reply:
오 좋네요. 다만 지금은 상수의 가시성을 변경해야하는데 이부분에 대해서 고민이 있네요 ㅎ
별도로 비지니스의 상수를 관리하는 파일을 만들거나 `.properties` 를 사용하는 것도 좋겠네요.